src/link.js

```diff
@@ -10,6 +12,25 @@ const copyAssetsIOS = require('./ios/copyAssets');
 
 log.heading = 'rnpm-link';
 
+const makeHook = (dependency, name) => (cb) => {
+  if (!dependency.config.hooks || !dependency.config.hooks[name]) {
+    return cb();
+  }
+
+  const hook = spawn(dependency.config.hooks[name], {
+    stdio: 'inherit',
+    stdin: 'inherit',
+  });
+
+  hook.on('close', function prelink(code) {
+    if (code) {
+      log.error(`Failed to link a "${dependency.name}" module`);
+    }
+
+    cb();
```

Comment:
Are you sure we want to swallow errors? NPM stops and I think it's reasonable for us to do either. Just cb(new Error())` should be enough with that `falied to link...` message instead of log.error call (it will also make the code to log from one place only)